docs/manual/mkdocs.yml

```diff
@@ -432,6 +432,8 @@ nav:
         - message remove: 'cmd/yammer/message/message-remove.md'
       - network:
         - network list: 'cmd/yammer/network/network-list.md'
+      - report:
+        - report groupactivityfilecounts: 'cmd/yammer/report/report-groupactivityfilecounts.md'
```

Comment:
Renamed from `groupactivityfilecounts` to `groupsactivitycounts` to match the issue description.